src/io/platform/f3xx/PWMf3xx.cpp

```diff
@@ -248,7 +248,7 @@ void PWMf3xx::setPeriod(float period) {
     // autoreload value into a valid range.
     do {
         prescaler++;
-        autoReload = period * clockFrequency / (prescaler + 1) - 1;
+        autoReload = clockFrequency / (prescaler + 1) - 1 / period;
```

Comment:
Now that we divide by the period here, we probably want to handle the case where someone enters 0 as a period